theme/svg_test.go

```diff
@@ -0,0 +1,31 @@
+package theme
+
+import (
+	"bytes"
+	"encoding/xml"
+	"image/color"
+	"io/ioutil"
+	"testing"
+
+	"fyne.io/fyne"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSVG_ReplaceFillColor(t *testing.T) {
+	src, err := ioutil.ReadFile("testdata/cancel_Paths.svg")
```

Comment:
I think that the SVG could be tested without needing the StaticResource below as you could get a Reader directly from the file.

Reply:
Yeah, not sure why I added that in the first place, fixed.